plugins/mongodb/database_credentials.go

```diff
@@ -0,0 +1,78 @@
+package mongodb
+
+import (
+	"fmt"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
```

Comment:
can we support the connection string you mentioned before as an optional field name as well?

then at provisioning time, we provision the connection string if it's set, and otherwise we provision the host and port, if they are set

Reply:
Good point! This is now handled in https://github.com/1Password/shell-plugins/pull/283/commits/9a19cd3ae4ba66339fa28d0b8a72ab254270df92.